dynawo/sources/Modeler/DataInterface/DYNShuntCompensatorInterface.h

```diff
@@ -118,6 +118,20 @@ class ShuntCompensatorInterface : public ComponentInterface {
    * @return @b true if the shunt compensator is linear, @b false otherwise
    */
   virtual bool isLinear() const = 0;
+
+  /**
+   * @brief Determines if voltage regulation is enabled on current shunt
+   *
+   * @return @b true if voltage regulation enabled, @b false if not
+   */
+  virtual bool isVoltageRegulationOn() const  = 0;
```

Comment:
I have the impression there is an additional space between const and = sign, can u remove it? thx

Reply:
OK